serialisation/src/main/java/org/ehrbase/serialisation/CanonicalJson.java

```diff
@@ -90,7 +90,7 @@ public String marshal(RMObject rmObject) {
         try {
             return JacksonUtil.getObjectMapper().readValue(value, clazz);
         } catch (IOException e) {
-            throw new UnmarshalException(e.getMessage(), e);
```

Comment:
Can you life that as UnmarshalException since that is the internal Exception of the serialisation  module